src/content/community/translations.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Translations
+---
+
+<Intro>
+
+React docs are translated by the global community into many languages all over the world.
+
+</Intro>
+
+## Main site {/*main-site*/}
+
+- [English](https://react.dev/) &mdash; [Contribute](https://github.com/reactjs/react.dev/)
+
+## Translated languages {/*translated-languages*/}
+
+{/* If you are a language maintainer and want to add your language here, finish the "Core" translations and edit `deployedTranslations` under `src/utils`. */}
+
+<LanguageList showTranslated={true} />
+
+## Languages undergoing translation work {/*languages-undergoing-translation-work*/}
+
+The list below includes languages with a significant amount of completed translation as well as those with little to no progress. The translation progress for each language is being tracked in [Is React Translated Yet?](https://translations.react.dev/)
+
+<LanguageList showTranslated={false} />
+
+## How to contribute {/*how-to-contribute*/}
```

Comment:
I think there's a link to a an issue or guide somewhere, can we add that?

Reply:
Maybe just link to this: https://github.com/reactjs/translations.react.dev

Reply:
This link is useful for those who want to start a new translation fork, but not for those who just want to work on translating individual articles. Each fork maintains its own contribution guide (I think), so let's direct contributors to each fork first.

Reply:
Yeah that makes sense